app/controllers/user/sessions_controller.rb

```diff
@@ -2,12 +2,15 @@ class User::SessionsController < ApplicationController
 
   def create
     user = User.find_by(email_address: params[:email_address])
-    if user.authenticate(params[:password])
+    if !user.nil? && user.authenticate(params[:password])
```

Comment:
Good catch since user.authenticate(params[:password]) will error out if user is nil